### PR TITLE
feat: 🤘🏼adding new GitHub actions

### PR DESCRIPTION
**Pull request by muenzpraeger**

### CODE REVIEW

**Comment by pozil** [comment]
How about adding a welcome message for PRs that reminds users to run prettier and tests?

**Comment by muenzpraeger** [comment]
Good point. Let’s discuss next week. I was wondering if we potentially should not only check Prettier - and instead run Prettier if the formatting is not correct, and commit back to a PR.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+### What does this PR do?
```
Comment (muenzpraeger) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: @pozil Check it out

Comment (pozil) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: This looks great!

**Comment by muenzpraeger** [comment]
We have to anyway till Monday, because of the auto-close of issues. ;-)

**Pull request merged**